docs/Workshop.md

```diff
@@ -0,0 +1,26 @@
+# Scaladays 2023 - TicTacToe
+
+The goal of this workshop is to develop a 2 player real-time strategy game using only Scala 3. In the process of this workshop, you will learn:
+
+* Scala 3
+* Scalajs
+* Functional Domain Modeling
+* Functional Reactive Programming in the style similar to [Elm](https://elm-lang.org/) using [Tyrian](https://github.com/purplekingdomgames/tyrian)
+* Kafka consumers and producers
+* Docker
+* docker-compose
+* sbt-native-packager
+* sbt tasks
+* Tagless Final
+* http4s with websockets
```

Comment:
It accepts modifications, but I think giving some sorting to the list would be good. In this case, I tried to move up the more general ones

```suggestion
* Functional Domain Modeling
* Functional Reactive Programming in the style similar to [Elm](https://elm-lang.org/) using [Tyrian](https://github.com/purplekingdomgames/tyrian)
* Message brokers with Kafka
* Tagless Final
* Scala 3
* ScalaJS
* Docker
* docker-compose
* sbt-native-packager
* sbt tasks
* http4s with WebSockets
```